plugin/kubernetes/object/service.go

```diff
@@ -20,7 +20,7 @@ type Service struct {
 }
 
 // ServiceKey return a string using for the index.
-func ServiceKey(name, namespace string) string { return name + "." + namespace }
+func ServiceKey(namespace, name string) string { return namespace + "/" + name }
```

Comment:
the built in keys are in the form `namespace/name`

Reply:
ok. I see reverse of params here..